WordPress/WordPressTest/ReaderTabItemsStoreTests.swift

```diff
@@ -139,36 +139,4 @@ class ReaderTabItemsStoreTests: XCTestCase {
             }
         }
     }
-
-    /// fetch request succeeds but type cast to [ReaderAbstractTopic] fails
-    func testGetItemsInvalidFetchedType() {
-
-        let mockTopics = ["we", "are", "not", "topics"]
```

Comment:
Removing this prevents crashes caused when fetching objects from Core Data with explicit types